### DISCUSSION TITLE
怎么使用puppeteer绕过Cloudflare的ddos防护

### DISCUSSION THREAD

**Discussion by axojhf** [DIYgod/RSSHub#10911] [Q&A] [answered]
比如这个网址：[https://warthunder.com/en/news/](https://warthunder.com/en/news/)，现在经常需要过cf的那个保护，新手不太懂怎么弄这种有cf保护的站点的rss

**Comment by TonyRL** [comment, marked as answer]
```js
await page.waitForSelector('.section');
```

**Reply by axojhf** [reply]
I try headless puppeteer, it always in cf protected page, but when I cancel headless, I can reach the target page before timeout.
I have added the `puppeteer-extra-plugin-stealth` library.
